File: week-7/stack.py
class Stack:
    def __init__(self):
        self._items = []

    def push(self, item):
        self.items.append(item)

    def pop(self):
        if not self.is_empty():
            return self.items.pop()
        return None

    def peek(self):
        if not self.is_empty():
            return self.items[-1]
        return None

    def is_empty(self):
        return len(self.items) == 0

    def size(self):
        return len(self.items)


stack = Stack()
stack.push(1)
stack.push(2)
stack.push(3)

stack.items[0] = 5

print(stack.items)  # Salida: [2]

File: week-7/stack.ts
class Stack {
  items: number[] = [];
}

const stack = new Stack();
stack.items.push(2);
stack.items.push(3);
stack.items.push(4);
stack.items.push(5);
stack.items.push(6);

// stack.items.pop();

// stack.items[0] = 1

stack.items.unshift(1);

console.log(stack.items);
